src/oca_github_bot/commands.py

```diff
@@ -53,14 +62,20 @@ def delay(self, org, repo, pr, username, dry_run=False):
 
 
 class BotCommandMerge(BotCommand):
-    bumpversion_mode = None  # optional str: major|minor|patch
+    bumpversion_mode = None
+    bumpversion_mode_list = ["major", "minor", "patch", "nobump"]
 
     def parse_options(self, options):
-        if not options:
-            return
```

Comment:
Why not raise RequirementOptionError here?

Reply:
because it will be raised after, if patch / minor / has not been found. no need to raise an error two times.

Reply:
ok. done.